content/blog/2023/when-to-choose-a-native-mobile-app.md

```diff
@@ -101,7 +109,16 @@ Native mobile apps live in app stores like the Google Play Store and Apple's App
 Store. If your research shows that the people you want to reach typically look
 for the type of content you will provide in app stores, or if you want to build
 your marketing plan around app stores, then a native mobile app is a better
-choice than a responsive web app (which is not listed in app stores).
+choice than a responsive web app which is not listed in app stores.
+
+On the other hand, a responsive web app with PWA enhancements can be listed in
+the app store. Additionally, building a marketing campaign around an app store
```

Comment:
Yep. I talk about how PWAs have to be wrapped in a native app (and mention PWABuilder) in the third article focused on PWAs. (If you're curious to read the third article, it's in Blog > 2023, but hidden.) I think it might be going into PWAs more than is necessary to mention that here as well.